apps/tx-builder/src/components/forms/fields/AddressContractField.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+import { ReactElement } from 'react';
+import { AddressInput } from '@gnosis.pm/safe-react-components';
+
+function AddressContractField({
+  id,
```

Comment:
You could spread a lot of these directly onto the component and not destructure them. It's not necessary but saves you a lot of hassle.